.github/pull_request_template.md

```diff
@@ -0,0 +1,145 @@
+
+
+---
+
+# Review Checklist
+
+## Reviewer: 
+
+### Implementation
+
+- [ ] Does this code change accomplish what it is supposed to do?
+- [ ] Can this solution be simplified?
+- [ ] Does this change add unwanted compile-time or run-time dependencies?
+- [ ] Was a framework, API, library, service used that should not be used?
+- [ ] Was a framework, API, library, service not used that could improve the solution?
+- [ ] Is the code at the right abstraction level?
+- [ ] Is the code modular enough?
+- [ ] Would you have solved the problem in a different way that is substantially better in terms of the code’s maintainability, readability, performance, security?
+- [ ] Does similar functionality already exist in the codebase? If so, why isn’t this functionality reused?
+- [ ] Are there any best practices, design patterns or language-specific patterns that could substantially improve this code?
+
+### Logic Errors and Bugs
+
+- [ ] Can you think of any use case in which the code does not behave as intended?
+- [ ] Can you think of any inputs or external events that could break the code?
+- [ ] Error Handling and Logging
```

Comment:
This should be a heading.

Reply:
Fixed in 6c75d54dd221700164ceb40b85a729e3547f8533.